src/pages/home/report/ReportActionItemSingle.js

```diff
@@ -67,10 +67,13 @@ const showUserDetails = (accountID) => {
 };
 
 function ReportActionItemSingle(props) {
-    const actorEmail = lodashGet(props.action, 'actorEmail', '').replace(CONST.REGEX.MERGED_ACCOUNT_PREFIX, '');
     const actorAccountID = props.action.actorAccountID;
-    const {avatar, displayName, pendingFields} = props.personalDetailsList[actorAccountID] || {};
-    const avatarSource = UserUtils.getAvatar(avatar, actorAccountID);
+    const isPolicyAction = ReportUtils.isPolicyExpenseChat(props.report) && !actorAccountID;
```

Comment:
I think this name is a bit misleading since this is not a reportAction with action = 'Policy'. Maybe `isWorkspaceActor` would be better?

Reply:
Great suggestion! Fixed.